plugins/janus_audiobridge.c

```diff
@@ -2851,25 +2957,27 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, int video, cha
 		participant->inbuf = g_list_insert_sorted(participant->inbuf, pkt, &janus_audiobridge_rtp_sort);
 		if(participant->prebuffering) {
 			/* Still pre-buffering: do we have enough packets now? */
-			if(g_list_length(participant->inbuf) == DEFAULT_PREBUFFERING) {
+			if(g_list_length(participant->inbuf) > DEFAULT_PREBUFFERING) { //If we have packets lost during prebuffering ... best to test > instead of == to stop prebuffering
 				participant->prebuffering = FALSE;
-				JANUS_LOG(LOG_VERB, "Prebuffering done! Finally adding the user to the mix\n");
+				JANUS_LOG(LOG_VERB, "(participant %"SCNu64") Prebuffering done! Finally adding the user to the mix\n", participant->user_id);
 			} else {
-				JANUS_LOG(LOG_VERB, "Still prebuffering (got %d packets), not adding the user to the mix yet\n", g_list_length(participant->inbuf));
+				JANUS_LOG(LOG_VERB, "(participant %"SCNu64") Still prebuffering (got %d packets), not adding the user to the mix yet\n", participant->user_id, g_list_length(participant->inbuf));
 			}
 		} else {
 			/* Make sure we're not queueing too many packets: if so, get rid of the older ones */
-			if(g_list_length(participant->inbuf) >= DEFAULT_PREBUFFERING*2) {
+			if(g_list_length(participant->inbuf) >= DEFAULT_PREBUFFERING*3) {
```

Comment:
Why change the multiplier here?

Reply:
We have actually some trouble associate to the size of inbuf (packet dropped); I have return the multiplier like before.